### ISSUE TITLE
Institution - managing collections information (wireframes)

### ISSUE THREAD

**Issue by mnwdeboer** [DiSSCo/ELViS#109]
#### Institution - managing collections information
For continuation of the Elvis application we would like to get feedback on the wireframes created for managing collection information of an institute. These collections wil show on the institute detail page and can also be used to be included in a VA/TA request.

These wireframes are based on the information from the xlsx file **'template for collections to include in VA call'**.
Because the documents are quit different from each other, the wireframes show two examples how to handle the different kinds of collection information. Below a wireframe a screenshot is shown on what xlsx it is based.

- **Wireframe 1**: Museum fuer Naturkunde has one collection which contains al disciplines.
- **Wireframe 2**: First you see the name of the collection with some additional information. Then you see the disciplines with there preservation options. An institute moderator can add additional disciplines to the collection.
- **Wireframe 3**: Museum National d'Histoire Naturelle shows multiple collections. More specified. See a xlsx example below in the wireframe
- **Wireframe 4**: Collection details of the 'Modern human remains' and the preservation details.

#### Wireframes
You can browse through the wireframes with the left/right arrow key.

https://invis.io/CVVDJZZ9J7A#/405189823_01-Institution-Collections-List-01

#### Question for this
Is this a good setup for managing collection information?

**Comment by FB-MfN** [comment]
Looks fine to me in general, but I'm not sure about the purpose of this setup in the system. Wouldn't it be an option to manage this kind of informatin through GirSciColl/CETAF/WikiData in order to avoid redundancy and to reduce efforts for keeping information up-to-date?

**Comment by cannedit** [comment]
Yes, we're aiming for this but in the second ELViS development phase.
